hash/sgi_mips.xml

```diff
@@ -1556,7 +1582,7 @@ license:CC0
 			<feature name="part_number" value="812-8101-008"/>
 			<!-- Origin: unknown -->
 			<diskarea name="cdrom">
-				<disk name="hot_mix_volume_8" sha1="ee725db3a52ff482e03b7806889763619f584859" />
+				<disk name="hot_mix_8" sha1="ee725db3a52ff482e03b7806889763619f584859" />
```

Comment:
You'll wanna drop the file renames as well

Reply:
Yeah, I thought I'd leave those since that's what the user selects/enters (and not necessarily linked to the name shown on the CD) and I think it's more intuitive if it's all similar. But if you want I'll drop those as well

Reply:
Yeah I'd rather not rename these. They won't show up in the UI afaik, it uses the description (for the set name) and the part name and part_id (if present) for the individual discs.